plugins/node/opentelemetry-instrumentation-redis-4/src/utils.ts

```diff
@@ -18,9 +18,10 @@ import {
   DbSystemValues,
   SemanticAttributes,
 } from '@opentelemetry/semantic-conventions';
+import { defaultDbStatementSerializer as redisSerializer } from '@opentelemetry/redis-common';
 
-export const defaultDbStatementSerializer: DbStatementSerializer = cmdName =>
-  cmdName;
+export const defaultDbStatementSerializer: DbStatementSerializer =
+  redisSerializer;
```

Comment:
Same here, no need to invove utils, you can import directly in `instrumentation.ts`